src/interpreter.js

```diff
@@ -143,13 +143,17 @@ function fileLocation(filename, lineno) {
 }
 
 function threadName(name) {
-  return ['<span class="threadName">', name, ' </span>'].join("")
+  return ['<span class="threadName">', padSpacesRight(name, 6), ' </span>'].join("")
 }
 
 function requestId(id) {
   return ['<span class="requestId">', id, ' </span>'].join("")
 }
 
+function keyVal(value) {
+  return ['<span class="key_val">', value, ' </span>'].join("")
```

Comment:
in modern browser you'd get much better performance by concating strings like this:
```
return `<span class="key_val">${value}</span>`
```
I think you should replace all `array.join` instances with this